dev/archery/archery/release.py

```diff
@@ -412,7 +412,7 @@ def curate(self):
                    if key not in within_keys]
 
         return ReleaseCuration(release=self, within=within, outside=outside,
```

Comment:
I would have expected `ReleaseCuration` to require a new field `minimal` defined. Line 244

Reply:
Yeah I thought so too but the fields are just set as `self._data` and passed through to the jinja renderer see [here](https://github.com/apache/arrow/blob/3830f05a2b6f354437b50e66f65a1578eebfc693/dev/archery/archery/utils/report.py#L32-L36) and are then available as jinja variables.